src/System.Memory/tests/ReadOnlyBuffer/ReadOnlySequenceTests.Common.cs

```diff
@@ -109,41 +109,49 @@ public void CheckEndReachableDoesNotCrossPastEnd()
         #region First
 
         [Fact]
-        public void CanGetFirst()
+        public void AsMultiSegment_CanGetFirst()
```

Comment:
Keeping AsMultiSegment_CanGetFirst() inside `ReadOnlySequenceTestsCommon<T>` abstract class.

But moved `AsString_CanGetFirst()` into:
```
public class ReadOnlySequenceTestsCommonChar: ReadOnlySequenceTestsCommon<char>
```
And moved `AsArray_CanGetFirst` and `AsMemoryManager_CanGetFirst` into:
```
public class ReadOnlySequenceTestsCommonByte: ReadOnlySequenceTestsCommon<byte>
```